### PR TITLE
Implement dbchangelog for financial transaction categories

### PR DESCRIPTION
**Pull request by Krisu93**

### CODE REVIEW

File: src/main/resources/db/changelog/005-create-table-financial_transactions_categories.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: "7"
+      author: Krisu93
+      changes:
+        - createTable:
+            tableName: financial_transactions_categories
```
Comment (mirekgab) [inline src/main/resources/db/changelog/005-create-table-financial_transactions_categories.yaml:7]: Please rename table to financial_transaction_categories.

**Review by odzio33 — APPROVED** [review]
Looks good for me

File: src/main/resources/db/changelog/006-insert-into-financial_transaction_categories.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: "8"
+      author: Krisu93
+      changes:
+        - loadData:
+            file: db/changelog/data/financial_transaction_categories.csv
+            tableName: financial_transactions_categories
```
Comment (mirekgab) [inline src/main/resources/db/changelog/006-insert-into-financial_transaction_categories.yaml:8]: This name should also be corrected.

**Review by mirekgab — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/data/financial_transaction_categories.csv [hunk 3]
```diff
@@ -0,0 +1,10 @@
+id,name,creation_date,transaction_type
+1,shopping,2023-02-23T23:47:52.595721658,EXPENSES
```
Comment (dawciobiel) [inline src/main/resources/db/changelog/data/financial_transaction_categories.csv:2]: According to `FinancialTransactionType` **[1]**, the financial transaction type is [ INCOME | EXPENSE ]. Similarly, the types of Financial Transaction Categories are the same.
These are singular nouns.

**[1]** https://github.com/bycza-zagroda/track-expenses-app-backend/blob/853bed3c414cb5e59332f21d944a507b727d3e9a/src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionType.java#L4

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Changes required

File: src/main/resources/db/changelog/data/financial_transaction_categories.csv (hunk 3 above)
Comment (Krisu93) [reply src/main/resources/db/changelog/data/financial_transaction_categories.csv]: Ok, so type of financial transactions in financial_transactions.csv is incorrect. I will correct this file right away

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Review by dawciobiel — APPROVED** [review]
Looks good for me

**Review by mirekgab — APPROVED** [review]

**Pull request merged**